lib/picker.js

```diff
@@ -233,7 +233,6 @@ function PickerConstructor( ELEMENT, NAME, COMPONENT, OPTIONS ) {
 
                 // Add the “active” class.
                 $ELEMENT.addClass( CLASSES.active )
-                aria( ELEMENT, 'expanded', true )
```

Comment:
jaws throws out an error if you have expanded on the trigger element, it should be used with the correct role.